src/Dto/OrganisationCsvUploadRequest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Dto;
+
+use App\Entity\OrganisationCsvUpload;
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+use Symfony\Component\Validator\Constraints as Assert;
+
+class OrganisationCsvUploadRequest
+{
+    /**
+     * @var UploadedFile
+     * @Assert\NotBlank(message="Please upload a valid csv")
+     * @Assert\File(
+     *     maxSize = "8M",
+     *     maxSizeMessage="The csv is too big to be uploaded, the maximum size is 8M",
```

Comment:
Just wondering why 8M. Is the server config done accordingly on docker?

Reply:
not especially